File: web_fund/algos/algo01.js
// 1. Get 1 to 255 - Write a function that returns an array with all the numbers from 1 to 255

function makeArray(num) {
    var arr = []
    for (var i = 0; i <= num; i++) {
        arr[i] = i + 1;
    }
    return arr;
}

console.log(makeArray(255))

File: practice_algos/python/algo01.py
# 1. Get 1 to 255 - Write a function that returns an array
# with all the numbers from 1 to 255

def get_1_to_255():
    arr = []
    for i in range(1, 256):
        arr.append(i)
    return arr
    
print(get_1_to_255())
